src/pages/AMM/Pools/components/PoolsTables/PoolsTable.style.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+import { theme } from '@/component-library';
+
+type StyledTablesWrapperProps = {
+  $hasPositions: boolean;
+};
+
+const StyledTablesWrapper = styled.div<StyledTablesWrapperProps>`
+  display: grid;
+  gap: ${theme.spacing.spacing6};
+  grid-template-areas: ${({ $hasPositions }) =>
+    $hasPositions
+      ? `'lend-positions-table'
+              'lend-assets-table'
+              'borrow-positions-table' 
+              'borrow-assets-table'`
+      : `'lend-assets-table'
+            'borrow-assets-table'`};
+
+  @media (min-width: 80em) {
```

Comment:
Should the `min-width` value be in the theme file?

Reply:
I actually do not need this file. Thank you.